tmt/steps/provision/testcloud.py

```diff
@@ -468,44 +423,90 @@ def start(self) -> None:
 
         # Create instance
         self.instance_name = self._tmt_name()
+
+        # Prepare DomainConfiguration object before Instance object
+        self._domain = DomainConfiguration(self.instance_name)  # type: ignore[name-defined]
+        self._domain.memory_size = self.memory * 1024
```

Comment:
How is `self.disk` used? I see `self._instance.disk_size = self.disk` being removed, just like `self._instance.ram = self.memory` is, but while the latter is replaced by the line above, the former does not seem to have a new form in the patch.

Reply:
@frantisekz can you check?